packages/bpk-docs/src/layouts/DefaultLayout/default-layout.scss

```diff
@@ -20,7 +20,21 @@
 
 .bpkdocs-default-layout {
   &__container {
-    background-color: $bpk-color-gray-100;
+    position: absolute;
+    display: flex;
+    width: 100%;
```

Comment:
This is necessary as some browsers (including IE and Edge) do not apply the flex properly in order to fill the browser window, making the webpage narrower than it should be